### PR TITLE
add step for help center in walkthrough

### PR DESCRIPTION
**Pull request by Eskibear**
part of #692 

@nickzhums @testforstephen @jdneo  please help to refine the description. 

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -160,6 +160,15 @@
               "image": "walkthrough/tutorials.png",
               "altText": "Tutorials"
             }
+          },
+          {
+            "id": "java.showHelpCenter",
+            "title": "Java Help Center",
+            "description": "Find more features and shortcuts in this page.\n[Open Java Help Center](command:java.welcome)",
```
Comment (nickzhums) [inline package.json:167]: Discover more shortcuts, and features on this page ?

Comment (Eskibear) [reply package.json:167]: ok, I'll take this.

**Comment by jdneo** [comment]
The screenshot is using `Java Tools` as the title of Java Help Center page, is it by design?

**Comment by Eskibear** [comment]
> is it by design?

I only changed webview title and command name which are "Welcome". Do you suggest to change title of the page?

**Comment by jdneo** [comment]
If originally it's called `Java Tools`, then I think it should be fine

**Review by jdneo — APPROVED** [review]

**Pull request merged**